pkgs/tools/security/safety/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, python3Packages }:
```

generally the interpreter is what you want to import, in case you need to freeze some dependency version, this is done at the interpreter level.
```suggestion
{ stdenv, python3 }:
```

---

the package set can be accessed with `python3.pkgs`